code/datums/ai/_item_behaviors.dm

```diff
@@ -0,0 +1,56 @@
+/datum/ai_behavior/item_escape_grasp
+
+/datum/ai_behavior/item_escape_grasp/perform(delta_time, datum/ai_controller/controller)
+	. = ..()
+	var/obj/item/item_pawn = controller.pawn
+	var/mob/M = item_pawn.loc
+	if(!istype(M))
+		finish_action(controller, FALSE) //We're no longer beind held. abort abort!!
+	item_pawn.visible_message("<span class='warning'>[item_pawn] slips out of the hands of [M]!</span>")
+	M.dropItemToGround(item_pawn, TRUE)
+	finish_action(controller, TRUE)
+
+
+
+/datum/ai_behavior/item_move_close_and_attack
```

Comment:
Document these new ai_behaviors, especially if you're already documenting their fields.